is_samples/is_5.3.0/hosting-endpoints/setup-accountrecovery.sh

```diff
@@ -33,27 +33,46 @@ unzip ${WEB_APP_HOME}/accountrecoveryendpoint.war -d ${WEB_APP_HOME}/accountreco
 
 rm ${WEB_APP_HOME}/accountrecoveryendpoint.war
 
-cp $IS_HOME/repository/components/plugins/org.wso2.carbon.base_4.4.11.jar $WEB_APP_LIB
```

Comment:
Since this PR applies for a later version better to create a different script under a different folder for that